.editorconfig

```diff
@@ -9,5 +9,10 @@ indent_style = space
 indent_size = 4
 trim_trailing_whitespace = true
 
+[*.{csproj,vcxproj}]
+indent_style = space
+indent_size = 2
+insert_final_newline = false
```

Comment:
:memo: This is the format Visual Studio uses when it makes changes to these files. Now hand-editing is consistent too.